### PR TITLE
fluent-bundle: Update `self_cell` to 1.x from 0.10.

### PR DESCRIPTION
**Pull request by waywardmonkeys**
This had no API changes.

### CODE REVIEW

**Review by alerque — APPROVED** [review]

**Comment by torokati44** [comment]
Just a quiet bump on this... :no_mouth:

**Comment by alerque** [comment]
It doesn't need a bump, I'll merge and address this as soon as we get a safe-harbor release tagged. Several folks have agreed not touching the default branch until that has been done is a good way to move forward. We're waiting on @zbraniecki to handle #317, then we'll move forward with stuff like this.

**Comment by torokati44** [comment]
Okay, I see, sorry then, and thanks for the update!

**Comment by alerque** [comment]
Slight overlap with #350 where I already queued up this bump, but I can take it out of there since it technically isn't a 'minor' bump and hence doesn't fit under that commit message anyway. Thanks for the contribution.

File: fluent-bundle/CHANGELOG.md [hunk 1]
```diff
@@ -18,7 +18,7 @@
   - Workspace: Add various missing documentation and fix typos and links
   - Workspace: Cleanup meta-data using workspaces, use SPDX licenses, etc.
   - Workspace: Apply rustfmt and clippy lint fixes
-
+
```
Comment (alerque) [inline fluent-bundle/CHANGELOG.md:21]:
```suggestion

```

**Review by alerque — APPROVED** [review]

**Pull request merged**